pkg/controller/kibana/config_reconcile.go

```diff
@@ -23,28 +23,43 @@ import (
 
 // Constants to use for the config files in a Kibana pod.
 const (
-	VolumeName        = "config"
-	VolumeMountPath   = "/usr/share/kibana/" + VolumeName
+	ConfigVolumeName                   = "elastic-internal-kibana-config-local"
+	ConfigVolumeMountPath              = "/usr/share/kibana/config"
+	InitContainerConfigVolumeMountPath = "/mnt/elastic-internal/kibana-config-local"
+
+	InternalConfigVolumeName      = "elastic-internal-elasticsearch-config"
+	InternalConfigVolumeMountPath = "/mnt/elastic-internal/kibana-config"
```

Comment:
`internal` is a bit confusing: why is `/mnt/elastic-internal/kibana-config` "internal" compared to `/mnt/elastic-internal/kibana-config-local`? Both living in `elastic-internal` sound like "internal" stuff.

Reply:
This is also to be consistent with what is done with Elasticsearch: 

```
    - mountPath: /mnt/elastic-internal/elasticsearch-config
      name: elastic-internal-elasticsearch-config
      readOnly: true
    - mountPath: /usr/share/elasticsearch/config
      name: elastic-internal-elasticsearch-config-local
```